### PR TITLE
(feat) Add configuration to disable empty sections

### PR DESCRIPTION
**Pull request by icrc-jofrancisco**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

As a developer, I want it to be able to give a configuration to disable the tabs without data showing in the UI.

## Screenshots

image

### CODE REVIEW

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx [hunk 1]
```diff
@@ -87,6 +87,13 @@ const VisitSummary: React.FC<VisitSummaryProps> = ({ encounters, patientUuid })
     encounters,
   ]);
 
+  const testsFilter = useCallback(() => {
```
Comment (ibacher) [inline packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx:90]: Does this actually work when tests are present? Normally, with `useCallback`, I'd expect to see the callback invoked somewhere, but the only use I see is on line 122 where we check the `length` property of the function, which should be [the number of arguments it takes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length).

**Review by ibacher — CHANGES_REQUESTED** [review]

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx (hunk 1 above)
Comment (gracepotma) [reply packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx]: @icrc-jofrancisco were you able to work on this change request?

**Comment by denniskigen** [comment]
Hey, @icrc-jofrancisco, could you rebase so I can merge this in.

File: packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx (hunk 1 above)
Comment (icrc-jofrancisco) [reply packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visit-summary.component.tsx]: > 

@denniskigen  should be ready for merge now. Thanks.

File: packages/esm-patient-chart-app/src/config-schema.ts [hunk 2]
```diff
@@ -32,45 +32,50 @@ export const esmPatientChartSchema = {
...
-  statusConceptSetUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the statuses for the queues.',
-    _default: '13c9b9fb-3396-4139-98e1-59938d544168',
-  },
-  defaultStatusConceptUuid: {
-    _type: Type.ConceptUuid,
-    _description: 'The UUID of the default status for the queues eg Waiting.',
-    _default: '136203AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+    _description: 'Disable notes/tests/medications/encounters tabs when empty',
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/config-schema.ts:38]:
```suggestion
    _description: 'Disable notes/tests/medications/encounters tabs when empty',
  },
```

Comment (denniskigen) [reply packages/esm-patient-chart-app/src/config-schema.ts:38]: Looks like you missed a closing brace on line 39.

**Review by denniskigen — CHANGES_REQUESTED** [review]

File: packages/esm-patient-chart-app/src/config-schema.ts (hunk 2 above)
Comment (icrc-jofrancisco) [reply packages/esm-patient-chart-app/src/config-schema.ts:38]: @denniskigen should be ready for merge now. Thanks.

File: packages/esm-patient-chart-app/src/config-schema.ts [hunk 3]
```diff
@@ -32,45 +32,51 @@ export const esmPatientChartSchema = {
...
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the statuses for the queues.',
+      _default: '13c9b9fb-3396-4139-98e1-59938d544168',
+    },
+    defaultStatusConceptUuid: {
+      _type: Type.ConceptUuid,
+      _description: 'The UUID of the default status for the queues eg Waiting.',
+      _default: '136203AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+    },
   },
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/config-schema.ts:80]: Hey @icrc-jofrancisco, it appears this unused brace is causing Prettier to fail. Please remove it and commit again.

Comment (icrc-jofrancisco) [reply packages/esm-patient-chart-app/src/config-schema.ts]: Hi @denniskigen , Done.

**Review by denniskigen — APPROVED** [review]
Thanks, @icrc-jofrancisco.

**Pull request merged**